aws/sdk/examples/ca-certs/src/main.rs

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_config::provider_config::ProviderConfig;
+use aws_smithy_client::erase::boxclone::BoxCloneService;
+use aws_smithy_client::hyper_ext;
+use rustls::RootCertStore;
+
+#[tokio::main]
+async fn main() {
+    let root_store: RootCertStore = todo!();
```

Comment:
Is this meant to be a `todo`? Shouldn't there at least be a comment telling people what to do?